scripts/rollup/build.js

```diff
@@ -176,26 +176,13 @@ function getBabelConfig(
   if (updateBabelOptions) {
     options = updateBabelOptions(options);
   }
+  // Controls whether to replace error messages with error codes in production.
+  // By default, error messages are replaced in production.
+  if (!isDevelopment && bundle.minifyWithProdErrorCodes !== false) {
+    options.plugins.push(require('../error-codes/transform-error-messages'));
```

Comment:
I hoisted this up because I noticed that the `noMinify` option was only being applied to the FB builds. So previously the `minifyWithProdErrorCodes` that I recently added was being ignored.

The effect of this PR is to stop minifying some error messages that were previously were minifying, like ones that run on the server.

The ones we actually care about, that run in the browser, are unaffected. But we now how this toggle to control precisely which builds to extract errors from.